### ISSUE TITLE
DavkySeznamRequest, DavkaZadostRequest, FunkcniMista

### ISSUE THREAD

**Issue by pacovskyvera** [nsessl/schema#36]
Obsahují Autorizaci – musel by se na obou stranách zavádět technický uživatel, jelikož to neprovádí žádný uživatel SSL.

**Comment by kosek** [comment]
A vadí to něčemu? Kvůli konzistenci mi přijde lepší to tam nechat.

**Comment by pacovskyvera** [comment]
Ano, jsou systémy SSL/ISSD které neumožnují zavádění technických uživatelů. Dále pak mnohé úřady využívají IDM pro správu uživatelů, která se používají např. na stránkách úřadu atd. Tedy pokud využijí technického uživatele musí ho zavést do IDM, to pro mnoho úřadů je nepřípustné.

**Comment by kosek** [comment]
U DavkySeznamRequest a DavkaZadostRequest jsem element Autorizace nastavil jako nepovinný.

U FunkciMistaRequest údaje o autorizace slouží jako parametr služby, takže tam jsem element Autorizace ponechal povinný.

Je to za vás takto OK?

**Comment by pacovskyvera** [comment]
Za nás ok. děkuji
